rootfs/api/models/app.py

```diff
@@ -185,8 +204,14 @@ def delete(self, *args, **kwargs):
         """Delete this application including all containers"""
         self.log("deleting environment")
         try:
-            # attempt to remove application from kubernetes
-            self._scheduler.destroy(self.id)
+            self._scheduler.delete_namespace(self.id)
+
+            # wait 30 seconds for termination
```

Comment:
little bit of column A, little bit of column B

It's 30 seconds + whatever latency each request adds :) each place has a `1` second sleep. I mostly copied this over from the scheduler but I can change the language if it is something we care about